src/main/java/build/buildfarm/common/redis/RedisMap.java

```diff
@@ -104,12 +108,32 @@ public void remove(JedisCluster jedis, Iterable<String> keys) {
    * @param jedis Jedis cluster client.
    * @param key The name of the key.
    * @return The value of the key. null if key does not exist.
+   * @note Overloaded.
    * @note Suggested return identifier: value.
    */
   public String get(JedisCluster jedis, String key) {
     return jedis.get(createKeyName(key));
   }
 
+  /**
+   * @brief Get the values of the keys.
+   * @details If the key does not exist, null is returned.
+   * @param jedis Jedis cluster client.
+   * @param keys The name of the keys.
+   * @return The values of the keys. null if key does not exist.
+   * @note Overloaded.
+   * @note Suggested return identifier: values.
+   */
+  public List<Map.Entry<String, String>> get(JedisCluster jedis, List<String> keys) {
+    JedisClusterPipeline p = jedis.pipelined();
+    List<Map.Entry<String, String>> values = new ArrayList(keys.size());
+    for (String key : keys) {
+      values.add(new AbstractMap.SimpleEntry<>(key, p.get(createKeyName(key)).get()));
```

Comment:
This has the effect of syncing for each item. I don't think this is what you want.

I recommend an Iterable response, constructed after a single sync, of a transform of a response set.

This will mostly devastate performance as is, as you're negating all of the purpose of the pipeline.

Reply:
what do you think about this change?  
I want to avoid returning the Response type to the caller since its a redis specific type.

Reply:
It's more correct, but looks really clunky. This should have a much cleaner collapse into an Iterable (return an iterable, and take an iterable as keys to boot) conventionally, rather than a List (the list interface connotes more than we want to imply here). Comprehend to a Map via a stream, then transform the map values and return its items.

Reply:
I've switched the input and return type to Iterable.  However, the implementation does not return a lazily list to the caller-- I think that's something to improve.